prisma/schema.prisma

```diff
@@ -94,7 +103,7 @@ model Access {
   updatedAt DateTime  @updatedAt
   userId    String
   projectId String
-  role      String?   @default("owner")
+  role      UserRole
```

Comment:
There is a migration missing for this default change

`ALTER TABLE "Access" ALTER COLUMN "role" DROP DEFAULT;`

If you reset your database and try running the migration/seed from scratch. You will notice.

Reply:
The newly generated column was dropped entirely and recreated, if you check on the migrations folder you'd see

Reply:
Ok I see it now, I'll work on that